device-types/Cisco/AIR-AP2802I-A-K9.yaml

```diff
@@ -0,0 +1,11 @@
+manufacturer: Cisco
+model: AIR-AP2802I-A-K9
+slug: air-ap2802i-a-k9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: gi0
```

Comment:
This should be the full interface name (i.e. `GigabitEthernet0`).

Reply:
Update interface name